pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/equation/lang/EquationParserTest.java

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.ddp.equation.lang;
+
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.broadinstitute.ddp.pex.lang.EquationLexer;
+import org.broadinstitute.ddp.pex.lang.EquationParser;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class EquationParserTest {
+    private static final String EXPRESSION = "3.14 + 2.71 * (x + y ^ 2)";
+    private static final String TREE = "(expression (expression (atom (number 3.14))) + " +
+            "(expression (expression (atom (number 2.71))) * (expression ( (expression (expression (atom (variable x))) + " +
+            "(expression (expression (atom (variable y))) ^ (expression (atom (number 2))))) ))))";
```

Comment:
This is a checkstyle violation- the `+` characters should be placed immediately after a newline, not before it.
```suggestion
    private static final String TREE = "(expression (expression (atom (number 3.14))) + "
            + "(expression (expression (atom (number 2.71))) * (expression ( (expression (expression (atom (variable x))) + "
            + "(expression (expression (atom (variable y))) ^ (expression (atom (number 2))))) ))))";
```

Reply:
Ok, will fix. Thanks